ql/src/test/queries/clientpositive/exim_04_evolved_parts.q

```diff
@@ -15,7 +15,7 @@ alter table exim_employee_n12 clustered by (emp_sex, emp_dept) sorted by (emp_id
 alter table exim_employee_n12 add partition (emp_country='in', emp_state='tn');
 
 alter table exim_employee_n12 set fileformat 
-	inputformat  "org.apache.hadoop.hive.ql.io.BucketizedHiveInputFormat" 
+	inputformat  "org.apache.hadoop.hive.ql.io.RCFileInputFormat"
```

Comment:
Why do we need this change?

Reply:
BucketizedHiveInputFormat can not be used this way, it is an internal inputformat used by mapjoins and it requires someone to setup either has.map.work or has.reduce.work property.
This test was passing because the partitions in this table are empty and the InputFormat.getSplits were not called for them in the FetchOperator. It is impossible to read from a table that has data in it and was setup like this.